src/utils/SystemDiagnostic.ts

```
@@ -0,0 +1,226 @@
+export interface SystemDiagnosticResult {
+    pulseAudioLatency: number
+    ffmpegBufferDelay: number
+    systemLoad: number
+    audioDeviceStatus: string
+    timestamp: number
+    variance: number
+}
+
+export class SystemDiagnostic {
+    private diagnosticHistory: SystemDiagnosticResult[] = []
+    private isRunning: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)